redisinsight/ui/src/pages/rdi/statistics/components/vertical-divider/VerticalDivider.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+import Divider from 'uiSrc/components/divider/Divider'
+
+import styles from './styles.module.scss'
+
+const VerticalDivider = () => (
```

Comment:
I'm not sure that we really need it. @romansergeenkosofteq  What do you think? i guess we can avoid additional component

Reply:
I'm good with using the Divider component directly instead. Just tried to make the usage more succinct.